dls-controls/malcolmjs
# Feature/text input local state #307

## Description

Added local state of textinput widget to redux store under attribute.localstate


## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA/layout/SEQ1
- edit the "prescale" attribute (or any other textinput widget) but DON'T press enter to submit
- click on the edit table button
- check that the local state is preserved when SEQ1 moves to the parent pane.

## Agile board tracking

connect to #307 
closes #307
